sdcm/cluster.py

```diff
@@ -2864,10 +2864,15 @@ def run_in_thread():
             start = current = time.time()
             while current - start < duration:
                 loader_node = random.choice(self.nodes)
-                db_node = random.choice(db_nodes)
-                self.log.info('Fullscan start on node {} from loader {}'.format(db_node.name, loader_node.name))
                 if loader_node.termination_event.isSet():
                     break
+                db_node = random.choice(db_nodes)
+                if db_node.remoter and not db_node.remoter.is_up():
+                    self.log.warning("Chosen node is down. Skip Request")
```

Comment:
why not choose another one?

Reply:
Fix. remove the sleep for timeout, which allow to restart and choose the db node again.